src/Inciter/Carrier.h

```diff
@@ -106,6 +107,7 @@ namespace inciter {
 
 extern ctr::InputDeck g_inputdeck;
 extern CkReduction::reducerType VerifyBCMerger;
+extern CkReduction::reducerType PDFMerger;
```

Comment:
Do we really want to add more global variables? Is there a reason this can't be passed as a parameter or as part of an object?